test/minikube/minikube_base_database_test.go

```diff
@@ -141,11 +141,18 @@ func verifyEngineAltAddress(t *testing.T, namespaceName string, admin0 string, e
 	}
 }
 
-func backupDatabase(t *testing.T, namespaceName string, podName string, databaseName string, options *helm.Options) {
+func backupDatabase(t *testing.T, namespaceName string, podName string, databaseName string, shouldTimeout bool, options *helm.Options) {
 
 	// wait for the backup to both start _and_ complete successfully
+	// if we expect backup to never schedule, don't use await. Instead sleep the backuptimeout and check the state of the pod using the same
+	backupTimeout := 120*time.Second
 	backupJob := fmt.Sprintf("hotcopy-%s-job-initial", databaseName)
-	testlib.AwaitPodPhase(t, namespaceName, backupJob, corev1.PodSucceeded, 120*time.Second)
+	if(!shouldTimeout) {
+		testlib.AwaitPodPhase(t, namespaceName, backupJob, corev1.PodSucceeded, backupTimeout)
+	} else {
+		testlib.AwaitPodPhase(t, namespaceName, backupJob, corev1.PodSucceeded, backupTimeout)
+	}
```

Comment:
this is the same code. It does not match what you wrote in the comment. What should it do?

Reply:
Removed, thx.